modules/openapi-generator/src/main/resources/typescript-fetch/sagaApiManager.mustache

```diff
@@ -1,15 +1,10 @@
-import {
-    Configuration,
-    ConfigurationParameters,
-} from "../";
+// Some imports not used depending on template conditions
+// @ts-ignore
+import { Configuration, ConfigurationParameters } from "../";
```

Comment:
**question:** Is there a particular schenario when any of these imports is not used?

Reply:
``ConfigurationParameters`` is not used if ``apiInfo`` and ``apis`` is not set.